cinq_auditor_required_tags/providers.py

```diff
@@ -125,11 +125,49 @@ def terminate_ec2_instance(client, resource):
             error
         ))
 
+def delete_s3_bucket(client, resource):
+    try:
+        session = get_aws_session(resource.account)
+        bucket = session.resource('s3', resource.location).Bucket(resource.resource_id)
+
+        lifecyclePolicy = {'Rules': [
+            {'Status': 'Enabled',
```

Comment:
Normally I'd ask for some or part of this to be a config item, but because it's basically the minimum requirement for a delete action I think it should be ok